src/cdk/portal/portal-directives.ts

```diff
@@ -93,13 +96,19 @@ export class CdkPortalOutlet extends BasePortalOutlet implements OnInit, OnDestr
     this._attachedPortal = portal;
   }
 
+  /** Component or view reference that is attached to the portal. */
+  get result(): ComponentRef<any> | EmbeddedViewRef<any> | null {
```

Comment:
Would it make more sense for the result to be emitted via `@Output` rather than directly read?

Reply:
I'm not sure whether that would be super convenient. People would still have to save a reference to it somewhere if they want to do anything with it.

Reply:
I think I'm being thrown off by calling it "result"; how about something like "attachedRef"?

Reply:
Would be flexible to have an `attached` output as well thinking about it.

Reply:
I like `attachedRef`. Initially I was going to do `instance`, but it would've gotten weird with the `ComponentRef.instance`. I've added the `attached` output as well.